services/outputhost/ackmanager.go

```diff
@@ -347,6 +353,13 @@ func (ackMgr *ackManager) updateAckLevel() {
 				default:
 				}
 			}
+			ackMgr.logger.WithFields(bark.Fields{
+				`status`:           oReq.GetStatus(),
+				`ackLevelAddress`:  oReq.GetAckLevelAddress(),
+				`ackLevelSeqNo`:    oReq.GetAckLevelSeqNo(),
+				`readLevelAddress`: oReq.GetReadLevelAddress(),
+				`readLevelSeqNo`:   oReq.GetReadLevelSeqNo(),
+			}).Info("ack level update succeeded")
```

Comment:
Yup, it will be noisy. May be instead of logging every update, I will log the ack manager state when the ackmgr is stopped.